api/worker_test.go

```diff
@@ -0,0 +1,50 @@
+package api
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/golang/mock/gomock"
+	"github.com/joschahenningsen/TUM-Live/mock_dao"
+	"github.com/joschahenningsen/TUM-Live/model"
+	"github.com/joschahenningsen/TUM-Live/tools"
+	"github.com/magiconair/properties/assert"
```

Comment:
Looks like a great opportunity to use https://github.com/stretchr/testify

Reply:
Right now, I am only using the` assert.Equals` function, do you have any particular features in mind?